### ISSUE TITLE
Commit dialog: Display commit output somewhere in the dialog

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#1400]
### Short Summary

For selfish reasons (kointegrate) I would like to see the output from git when I make a commit in the commit dialog.  
Sample output:
```
C:\Users\careyh\Projects\Komodo\KomodoIDE>git commit -a
[master 91bc5a9] prefs: commando prefs missing padding fixes #1388
 1 file changed, 133 insertions(+), 133 deletions(-)
 rewrite src/modules/commando/content/ui/pref.xul (82%)
```
That's quite a bit of output and it can be a small commit pane dialog.  Maybe it can scale the info down as the dialog shrinks, or just trim most of that info down by default.

Just in case this pops into someones head like it did for me; the current dialog that appears (unless you told it not to) that ask if you want to see the command out is misleading and not a good UX:
1. It's jarring to have that dialog appear and ask for action (yes you can get rid of it but why have it in the first place if you can provide the output any way in an appropriate, non intrusive manner...see point 3)
2.  Even though it says it's going to show you command output, it doesn't actually show you anything having to do with git and it's out put.  It shows you the command that Komodo just ran and that's it.  That's not helpful
3. It pulls you out of the commit pane (if you used that) and drops your focus in the notifications pane.  That's jarring as well.

SOOOO, the new UX should replace the old UX basically.  Man that was long winded.

**Comment by Naatan** [comment]
Yeah I agree the command output stuff is definitely not a good UX. I didn't have enough time to rework that.

My intention was (is) to use the Log tab in the version control widget to show more useful data, but I like the idea of showing in right in the commit "dialog" too. We could show it in the area that it shows the diff. So when you commit the diff changes to the command output and then when you make new selections it shows the diff again.

**Comment by cgchoffman** [comment]
> So when you commit the diff changes to the command output and then when you make new selections it shows the diff again.

This makes sense.  I don't know what the term for this but the output from git is "convenient" information.  It's good for a quick references but it doesn't need to be saved.  If you need information from it again you can get it from `git log` in the commit pane.
